src/osara.h

```diff
@@ -317,10 +318,12 @@ typedef enum {
 	TF_MEASURE,
 	TF_MINSEC,
 	TF_SEC,
+	TF_ROUNDSEC,
 	TF_FRAME,
 	TF_HMSF,
 	TF_SAMPLE,
-	TF_MEASURETICK
+	TF_MEASURETICK,
+	TF_MILSEC
```

Comment:
Rename TF_MILSEC to TF_MS.
```suggestion
	TF_MS
```